src/main/scala/sigmastate/serialization/ValueSerializer.scala

```diff
@@ -115,9 +115,17 @@ object ValueSerializer extends SigmaSerializerCompanion[Value[SType]] {
     serializer
   }
 
+  private val ConstantPlaceholderIndexCode = ConstantCode
```

Comment:
@ergomorphic Not so sure about this one. We might need a separate code to denote an upcoming constant placeholder index.

Reply:
@greenhat yes it is better no have different codes